test/libsolidity/semanticTests/cleanup/cleanup_bytes_types_shortening_newCodeGen.sol

```diff
@@ -5,13 +5,13 @@ contract C {
         assembly {
             r := y
         }
-        // At this point, r and y both store four bytes, but
+        // At this point, r stores four bytes and y stores 2 bytes
         // y is properly cleaned before the equality check
         require(y == bytes2(0xffff));
```

Comment:
Haha, sorry for this apparently being confusing :-). But the idea is to keep the *original*, *unmodified* comment and ``require`` in the ``OldCodeGen`` file and *not* have those three linkes in the ``newCodeGen`` file :-).